app/features/send/send-scanner.tsx

```diff
@@ -0,0 +1,99 @@
+import {
+  PageBackButton,
+  PageContent,
+  PageHeader,
+  PageHeaderTitle,
+} from '~/components/page';
+import { QRScanner } from '~/components/qr-scanner';
+import { useExchangeRate } from '~/hooks/use-exchange-rate';
+import { useToast } from '~/hooks/use-toast';
+import type { Money } from '~/lib/money';
+import { useNavigateWithViewTransition } from '~/lib/transitions/view-transition';
+import type { Account } from '../accounts/account';
+import { useSendStore } from './send-provider';
+
+/**
+ * Converts an amount to the send account currency.
+ * If the amount is already in the send account currency, returns the amount.
+ *
+ * @throws if the exchange rate fails to load
+ */
+const useConverter = (sendAccount: Account) => {
```

Comment:
I am not sure I like the way converion is done here but need to think a bit more